script/tool/lib/src/common/plugin_utils.dart

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:file/file.dart';
 import 'package:flutter_plugin_tools/src/common/repository_package.dart';
+import 'package:pubspec_parse/pubspec_parse.dart';
```

Comment:
Maybe the `repository_package.dart` should reexport the `Pubspec` from `package:pubspec_parse` so people don't need this import to use `parsePubspec()`?

Reply:
For some reason `export`ing within internals isn't something I think about. Thanks for the suggestion!

Cleaned up a lot of existing imports as part of this.